python/ray/data/_internal/metadata_exporter.py

```diff
@@ -138,20 +142,28 @@ class DatasetMetadata:
     data_context: DataContext
 
 
-def sanitize_for_struct(obj):
+def _add_ellipsis(s, truncate_length):
```

Comment:
Nit: This name sounds like it always add ellipsis, but it's more like `_truncate_string`